docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -309,6 +309,18 @@ focus for this specification. More specifically, on non-EFI systems
 configuration snippets following this specification cannot be used to spawn
 other operating systems (such as Windows).
 
+Unfortunately, there are implementations of boot loading infrastructure that
+are also using the /loader/entries/ directory, but place files in them that are
+not valid by this specification. This is quite unfortunate. In order to
+minimize confusion a boot loader implementation may place a file
+/loader/entries.srel next to the /loader/entries/ directory containing the
+ASCII string "standard" (suffixed with a UNIX newline). Tools that need to
```

Comment:
So I really would avoid the abbreviated "bls" moniker. It's not mentioned even a single time on https://systemd.io/BOOT_LOADER_SPECIFICATION but it *is* mentioned a lot on https://fedoraproject.org/wiki/Changes/BootLoaderSpecByDefault

Hence I would just give up the short abbreviation "bls" — it's more associated with the fedora/grub thing than with ours stuff. Given that the feature is precisely about distinguishing our stuff from theirs I think it would be a really bad idea to pick the moniker they so far squatted. That just adds more confusion.

I am OK with using basically any moniker for this, but "bls" is the worst choice I think.

I also would like the spec to not carry the systemd name in anything, given it's generic, and if Grub one day maybe implements our upstream spec they shouldn't claim their implementation was "systemd", hence I also wuld not put the "systemd" string in that file.

hence, if not "bls", and not "systemd", what else? "fdo" could work, but we moved aaway from that for new stuff. "official", "upstream", "original" all came to mind, but I figured "standard" is a nicer thing, since it's not the fedora-specific hack but the "standard" version, i.e. the implemenation is more than just a specific implementation by fedora, but more cross-platform...

Anyway, I am not glued to "standard", all I know it shouldn't be "bls", and probably shouldn't be "systemd" either. ideas?

Reply:
To maybe add to the bikeshedding: How about a long name like `io.systemd.bls` (or `io.systemd.boot_loader_specification`). It would make clear which standard (or official, upstream or original version) it is and it would nicely delineate it from something like `org.fedora.bls` and implementors could say "we faithfully implement `io.systemd.bls`" or "our flavour is `org.grub.bls`". Not that I would expect a great many implementations, but with the Fedora stuff there's two already.

Reply:
@behrmann as mentioned, I'd prefer a string that doesn't include the word "systemd", so that we make it a bit politically easier for grub or so to implement this. i.e. I think it's easier for some people to swallow this if there is no pesky reference to systemd in the stuff placed in the ESP...

Reply:
So I guess we can put the string "type1" in there. I think that would work out nicely, no?

Reply:
I understand the political argument, although I fear this will be an uphill battle as long as this isn't enshrined in POSIX regardless of how this is named, but I guess you're right since some things like `/etc/os-release` have already filtered through as well. 

I think I like "type1" better than the rest, since it sounds less definitive and more inviting for others to use this, although it leaves open who can make up new ones and how they are to be enumerated, that was what I was trying to get at with the more qualified name. I also think that it would make sense to avoid bls, so as to avoid possible confusions with the extended stuff in Fedora.

Reply:
I'd vote for `type1` or `bls-type1`.